### ISSUE TITLE
Feature: Set a DPI value for exporting plots

### ISSUE THREAD

**Issue by batmel** [highcharts/highcharts#7337]
#### Expected behaviour
The possibility to set the width and height for a plot **and** a DPI number, to make the downloaded plot useable for various cases.
This is especially useful if you need plots for scientific contributions, papers, thesis ...

#### Actual behaviour
So far you can only set a width, height and a scaling factor. You cannot set the dpi.

**Comment by KacperMadej** [comment]
Hi,

Thank you for suggesting the enhancement.

Number of DPI depends on screen that you are using to view an exported image - on mobile DPI will be higher, on full HD 50 inch screen DPI will be lower, on [computer monitors it's not standardized](https://en.wikipedia.org/wiki/Dots_per_inch#Computer_monitor_DPI_standards).

If you are preparing an image for 300 DPI, then chart with 300 pixel width and height will have size of 1 square inch. If you set width and height then size depends on screen / printer DPI.

Could you provide more info?

**Comment by batmel** [comment]
@KacperMadej of course. Pixels, DPI and print size are obviously related. Pixels = Resolution(DPI) x Size.

If you want to export a plot as .png or .jpg, that needs to have a specific width, due to space restriction, for example 90mm/3.54 inch, but still need to have a good resolution, it would be nice to set the DPI value to 300 to get in the end 1063 pixels.

So adjustable DPI would allow high-quality images of smaller physical page. This should prevent that the image quality suffers, due to small image sizes.

**Comment by TorsteinHonsi** [comment]
@batmel Thanks for your suggestion! You're aware of the [exporting.scale](https://api.highcharts.com/highcharts/exporting.scale) option? It allows you to get the resolution you want, though it won't tell the printer how to handle it.
